exampleCourse/questions/demo/autograder/python/randomFunctionName/question.html

```diff
@@ -0,0 +1,42 @@
+<pl-question-panel>
+    <p>
+        <i>
+            This is a small example question that shows how we can randomize questions using the Python autograder.
+            <code>server.py</code> randomizes a function name and a list of input/output pairs the function must
+            correctly handle, plus a default output for every other case. The test file checks these pairs and the default output.
+        </i>
+    </p>
+{{#params}}
```

Comment:
Clever, though for an example question, I'd rather have explicit `{{params.function_name}}` etc.

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/7903/commits/a08effde0951603eecad2156fea3d48d46b4941c